chia/util/service_groups.py

```diff
@@ -39,7 +40,9 @@ def all_groups() -> KeysView[str]:
 
 def services_for_groups(groups: Iterable[str]) -> Generator[str, None, None]:
     for group in groups:
-        yield from SERVICES_FOR_GROUP[group]
+        service = SERVICES_FOR_GROUP[group]
+        if service:
+            yield from service
```

Comment:
If I am reading this correctly, we now avoid yielding from empty groups.  But they were empty anyways so nothing would have been yielded.  Is this important to special case?

Reply:
This is a code to avoid previous `mypy` error but currently no reason for this change. I reverted this change.